extras/zot/enable-zot/enable-zot.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: zot-cache-enable
+spec:
+  rules:
+  - name: clusterconfig
+    context: 
+       - name: userconfig
+         configMap:
+           name: "{{ request.object.spec.userConfig.configMap.name }}"
+           namespace: "{{ request.object.spec.userConfig.configMap.namespace }}"
+    match:
+      any:
+      - resources:
+          kinds:
+            - App
+          namespaces:
+            - 'org-*'
+          operations:
+            - CREATE
+    preconditions:
+      all:
+        - key: "{{ request.object.spec.catalog }}"
+          operator: Equals
+          value: cluster
+        - key: "{{ request.object.spec.name }}"
+          operator: Equals
+          value: cluster-aws
```

Comment:
This will only target `cluster-aws` - would it not be better to expand the pre-conditions for other cluster type even if zot is not immediately enabled for those cluster types?